First draft version of the proposal (#6)
UBC-MDS/DSCI_532_L02_group207_dashboards: proposal.md

Example:

> > Barley is part of the major cereal grains used worldwide. Understanding how the variety of barley and the site on which they grow impact the annual yield is very important. It can help farmers to have the highest yield as possible and increase their productivity, but is can also help agronomists to have a better understanding of how the environment impacts each variety of barley. To help those farmers increasing their productivity, we decided to create an app that could allow them to explore a dataset containing information on annual yields for different sites and barley varieties. This app will allow the users to see the annual yield for selected varieties and particular sites, for the years 1931, 1932, or both. It should help them understand what variety or what site is the more suitable to their situation.

Reviewer: ```suggestion
> Barley is part of the major cereal grains used worldwide. Understanding how the variety of barley and the site on which it grows impacts the annual yield is very important. It can help farmers to have the highest yield as possible and increase their productivity, but it can also help agronomists to have a better understanding of how the environment impacts each variety of barley. To help those farmers increase their productivity, we decided to create an app that could allow them to explore a dataset containing information on annual yields for different sites and barley varieties. This app will allow the users to see the annual yield for selected varieties and particular sites, for the years 1931, 1932, or both. It should help them understand what variety or what site is the most suitable to their situation.
```

Reviewer: Just some suggested changes regarding grammar/spelling. Please feel free to revert if you disagree!

Example:

> > We chose the barley dataset from the vega-datasets python package. This dataset shows the yields of 10 different varieties of barley at 6 different sites in Minnesota during the years 1931 and 1932. It first appeared in 1934 the paper "Statistical Determination of Barley Varietal Adaption" by the three agronomists who leaded this experiment : F.R. Immer, H.K. Hayes, and L. Powers.

Reviewer: ```suggestion
We chose the barley dataset from the vega-datasets python package. This dataset shows the yields of 10 different varieties of barley at 6 different sites in Minnesota during the years 1931 and 1932. It first appeared in the 1934 paper  "Statistical Determination of Barley Varietal Adaption" by the three agronomists who led this experiment: F.R. Immer, H.K. Hayes, and L. Powers.

```

An example usage scenario with tasks (tasks are indicated in brackets, i.e. [task])

> Our research questions are : 
Given some sites and some varieties, what variety of barley had the highest yield during a specific year?
Given some sites and some varieties, what site had the highest yield during a specific year?
Given some sites and some varieties, what is the variety of barley with the highest yield for each of the sites?

> > George is a farmer who just bought this land in Waseca and who already has a land in Crookston. He would like to grow barley on his two lands, but he doesn't know which variety he should use to have the highest yield. 

Reviewer: ```suggestion
> George is a farmer who just bought land in Waseca and who already has land in Crookston. He would like to grow barley on his two lands, but he doesn't know which variety he should use to have the highest yield. 
```
